app/modules/encounters/resources.py

```diff
@@ -176,19 +96,61 @@ def get(self, encounter):
     )
     @api.login_required(oauth_scopes=['encounters:write'])
     @api.parameters(parameters.PatchEncounterDetailsParameters())
-    @api.response(schemas.DetailedEncounterSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def patch(self, args, encounter):
         """
         Patch Encounter details by ID.
         """
+
+        edm_args = [
+            arg
+            for arg in args
+            if arg.get('path')
+            in parameters.PatchEncounterDetailsParameters.PATH_CHOICES_EDM
+        ]
+        if edm_args and len(edm_args) != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(400, 'Cannot mix EDM patch paths and houston patch paths')
+
+        if edm_args:
+            log.debug(f'wanting to do edm patch on args={args}')
+            try:
+                result_data = current_app.edm.request_passthrough_result(
+                    'encounter.data',
+                    'patch',
+                    {'data': args},
+                    encounter.guid,
+                    request_headers=request.headers,
+                )
+            except HoustonException as ex:
+                log.warning(f'EDM patch got {ex}')
+                abort(400, ex.message, error=ex.error, edm_status_code=ex.edm_status_code)
```

Comment:
any reason you are using 400 rather than ex.code?

Reply:
i am not 100% sure what the best course of action are for passing error codes as-is from edm.  i actually realize it probably _does_ make sense to pass through some (e.g. 404 certainly seems an obvious one) .. but others, just like generic death ones like 500 ...  not sure.  very likely, frontend is only going to care about 200 or not-200.... i might flip this to pass it through as-is, especially now that i am thinking about 404.   besides, the ones that flask definitely dislikes (>600) is getting absorbed and turned into 400 _already_ inside houston edm extension code.